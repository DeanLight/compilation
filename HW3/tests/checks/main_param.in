void main (int param)
{
	return;
}