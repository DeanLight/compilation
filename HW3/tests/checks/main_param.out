---end scope---
param INT -1
Program has no 'void main()' function
